### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR we created a test for all the components of the website.
Thanks for the review.

### CODE REVIEW

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  });
+
+  it('When a AC key is pressed', () => {
+    const obj = {
+      equalSign: true, next: null, operation: null, total: '13',
+    };
+    const newObj = { next: null, operation: null, total: null };
+    const button = 'AC';
+    expect(calculate(obj, button)).toEqual(newObj);
+  });
```
Comment (bellom) [inline src/__tests__/calculate.test.js:47]: We need to write a unit test for all the functionalities & scenarios in `calculate.js` module.

We are missing unit test for the following test case:
- `buttonName === '.'`
-  `buttonName === '='`
-  `buttonName === '+/-'`

**Review by bellom — COMMENTED** [review]
Review In progress...

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well writing almost all the unit test and jest snapshots, good job setting up linter and good commit messages except for one commit message that you did not capitalize 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments above the review._

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/calculate.test.js (hunk 1 above)
Comment (Mhdez221993) [reply src/__tests__/calculate.test.js:47]: Thank you for review our project 👍

Comment (henatan99) [reply src/__tests__/calculate.test.js:47]: - This issue is well addressed. You did an amazing job! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @Leandro-Barretoo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] I think you can update your readme with the addition of the test running command in the setup section. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**